doc/how_to/styling/altair.md

```diff
@@ -0,0 +1,56 @@
+# Style Altair Plots
+
+This guide addresses how to style Altair plots displayed using the [Vega pane](../../../examples/reference/panes/Vega.ipynb).
```

Comment:
Please check if this will work when rendered for all the `.md` files.

I have not been able to find links from docs to reference guides in the existing documentation, so I don't know if this will work.